Area51/Sources/TableViewCells/ListingTableViewCell.swift

```diff
@@ -1,12 +1,55 @@
+import ImageService
```

Comment:
Preferred two cells because we did not need extra logic about hiding and not hiding UI. As a cell gets more complex, this logic will be more complex. Also having a variety of cells will allow developers to easily add new ones to try out new layouts.

Reply:
I agreed. I figured for this simple case it would be okay to do this. If it gets more complex we can always create a new cell(s)

We don’t know off hand right now how complex the cell will get and when we get there we can make that decision?